app/src/main/java/com/github/geohunt/app/LoginActivity.kt

```diff
@@ -27,10 +27,21 @@ import com.github.geohunt.app.ui.theme.seed
 import com.github.geohunt.app.utility.replaceActivity
 
 class LoginActivity : ComponentActivity() {
+    private val PREFERENCES_FILE = "preferences"
```

Comment:
You don't need a name and could just use the getPreferences() function as you don't seem to share this file between activities.
Or you should add this name to the string ressources to make it accessible to everyone if that's the point.
By the way if this preference file is only going to get used to know if the user already saw the tutorial, you might want to change the name as preferences doesn't really reflect this.

Reply:
The information about "preferences" should be maintainable in the context of the future code. Therefore, to leave the code modular, this constant should remain as it is.

I have therefore added a string for preferences to the strings.

Reply:
I was thinking of the constants that were already defined by shared preferences interface (see https://developer.android.com/reference/android/content/Context#getSharedPreferences(java.lang.String,%20int)) but I guess it's ok